deploy/rbd/kubernetes/csi-attacher-rbac.yaml

```diff
@@ -10,9 +10,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rbd-external-attacher-runner
 rules:
-  - apiGroups: [""]
-    resources: ["events"]
-    verbs: ["get", "list", "watch", "update"]
```

Comment:
can you add this too?
https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/rbac.yaml#L31